docs/framework/wcf/how-to-create-a-wcf-client.md

```diff
@@ -1,66 +1,125 @@
+> If you call a WCF service from a class library project in Visual Studio, use the **Add Service Reference** feature to automatically generate a proxy and associated configuration file. However, because class library projects don't use this configuration file, you need to add the settings in the generated configuration file to the *App.config* file for the executable that calls the class library.
 
 > [!NOTE]
-> When calling a WCF service from a class library project in Visual Studio, you can use the **Add Service Reference** feature to automatically generate a proxy and associated configuration file. The configuration file will not be used by the class library project. You need to add the settings in the generated configuration file to the app.config file for the executable that calls the class library.
+> As an alternative, use the [ServiceModel Metadata Utility tool](#servicemodel-metadata-utility-tool) instead of Visual Studio to generate the proxy class and configuration file.
 
-The client application uses the generated proxy class to communicate with the service. This procedure is described in [How to: Use a Client](../../../docs/framework/wcf/how-to-use-a-wcf-client.md).
+The client application uses the generated proxy class to communicate with the service. This procedure is described in [Tutorial: Use a client](how-to-use-a-wcf-client.md).
 
-## To create a Windows Communication Foundation client
+In this tutorial, you learn how to:
+> [!div class="checklist"]
+> - Create and configure a console app project for the WCF client.
+> - Add a service reference to the WCF service to generate the proxy class and configuration files.
 
-1. Create a new console application project in Visual Studio. Right-click on the Getting Started solution in **Solution Explorer** and select **Add** > **New Project**. In the **Add New Project** dialog, on the left-hand side, select the **Windows Desktop** category under **Visual C#** or **Visual Basic**. Select the **Console App (.NET Framework)** template, and then name the project **GettingStartedClient**.
 
-2. Add a reference to System.ServiceModel to the GettingStartedClient project. Right-click on the **References** folder under the GettingStartedClient project in **Solution Explorer**, and then select **Add Reference**. In the **Add Reference** dialog, select **Framework** on the left-hand side of the dialog under **Assemblies**. Find and select **System.ServiceModel**, and then choose **OK**. Save the solution by selecting **File** > **Save All**.
+## Create a Windows Communication Foundation client
 
-3. Add a service reference to the Calculator Service.
+1. Create a console app project in Visual Studio. 
 
-   1. First, start up the GettingStartedHost console application.
+    1. In the **Solution Explorer** window, select the **GettingStarted** solution (top node), and then select **Add** > **New Project** from the shortcut menu. 
```

Comment:
I'll change shortcut menu if you really want, but we've decided that it's better than using "right-click" (which we've been instructed to avoid). Also, "shortcut menu" is an accepted UI term in the style guide: https://worldready.cloudapp.net/Styleguide/Read?id=2700&topicid=26472